beater/journalbeat.go

```diff
@@ -293,7 +286,7 @@ func (jb *Journalbeat) Run(b *beat.Beat) error {
 			})
 
 			if err != nil {
-				logp.Err("Metrics collection for log processing on this host is disabled")
```

Comment:
We usually do ("... %v", err), in case the error has any extra fields, that might be relevant for troubleshooting.